website/docs/secrets/setup-secrets-operator.mdx

```diff
@@ -3,12 +3,142 @@ title: Setup Secrets Operator
+
+### Setup SOPS on management cluster or existing leaf cluster
+
+To setup SOPS on management or existing leaf cluster, follow this [guide](/secrets/managing-secrets-with-sops.mdx).
+
+import SopsBootstrapJob from "!!raw-loader!./assets/sops-bootstrap-job.yaml";
+import TemplateParams from "!!raw-loader!./assets/template-params.yaml";
+import TemplateAnnotations from "!!raw-loader!./assets/template-annotations.yaml";
+
+### Bootstrapping SOPS to leaf clusters
+
+Bootstrapping SOPS to leaf clusters in WGE can be done by utilizing `ClusterBootstrapConfig` job to bootstrap Flux and SOPS.
+The job is a container which generates SOPS secrets key pair, creates a kubernetes secret with the private key, creates a kubernetes secret with the public key (to be used in self-serve flow) and the proper rbac for it.
+As well as an option to push the public key to the git repository via a PR (to be distributed).
+
+### Prerequisites
+
+#### ClusterBootstrapConfig job
+
+The following example is using GPG encryption to install SOPS and generate keys when bootstrapping leaf clusters. Create the following `ClusterBootstrapConfig` CR and push it to your fleet repo.
+        
+<details><summary>Expand to view </summary>
+    <CodeBlock
+    title="clusters/management/capi/boostrap/sops-bootstrap-job.yaml"
+    className="language-yaml"
+    >
+        {SopsBootstrapJob}
+    </CodeBlock>
+</details>
+
+#### Cluster template updates
+
+In order to bootstrap SOPS to leaf clusters, we need some modifications to the cluster template to allow creating a [Kustomization](https://fluxcd.io/flux/guides/mozilla-sops/#configure-in-cluster-secrets-decryption)
+for reconciling the secrets on the cluster using SOPS and to run the `ClusterBootstrapConfig` job during cluster creation.
+
+The template metadata should have annotation, it will be used by WGE to create the Kustomization with the cluster files.
+
+```yaml
+templates.weave.works/sops-enabled: "true"
+```
+
+The template should have the following parameters that are needed for the Kustomization
+
+<details><summary>Expand to view </summary>
+    <CodeBlock
+    title="clusters/management/capi/templates/template.yaml"
+    className="language-yaml"
+    >
+    {TemplateParams}
+    </CodeBlock>
+</details>
+
+The template should have the following annotations under `GitOpsCluster` to be used in the bootstrap job
+
+<details><summary>Expand to view </summary>
+    <CodeBlock
+    title="clusters/management/capi/templates/template.yaml"
+    className="language-yaml"
+    >
+    {TemplateAnnotations}
+    </CodeBlock>
+</details>
+
+### Installation Steps
+
+To bootstrap SOPS on a leaf cluster, create a new cluster using the SOPS template from the `Create Cluster` page.
```

Comment:
I am sorry if I mistakenly said to remove the create cluster screenshot with the highlighted fields. Can we add it back in?